### ISSUE TITLE
Descompile UModel and UPoly from AVolumes in A Hat in Time

### ISSUE THREAD

**Issue by penguin21** [EliotVU/Unreal-Library#87]
I am trying to decompile the data from the AVolumes with the UPolys and the UModels and the output window appears blank, and I am using the UE Explorer source code but the numbers appear negative in the Polys Array.

Screenshots:
UPolys Output:
![image](https://github.com/user-attachments/assets/6e185221-d23d-4f35-894c-ba9d2685cc23)

UModel Output:
![image](https://github.com/user-attachments/assets/6b76215d-bb75-44e8-97c1-961ecdc67aab)
Error in debugger when you try UPolys:
![image](https://github.com/user-attachments/assets/2481c613-ffa6-4d81-a5b9-7d40daaf2c88)

**Comment by EliotVU** [comment]
Afaik the error is caused by changes made to UPolys for A Hat in Time (but I haven't verified this)

**Comment by EliotVU** [comment]
It should no longer throw an error, but the content being empty is expected behavior a.t.m

**Comment by EliotVU** [comment]
That said, I'll see if I can back port the T3D exporter for this class.

**Comment by penguin21** [comment]
That means right now the aplication now can read the UPolys but can't show in the output right now?

**Comment by EliotVU** [comment]
Yes that is right, UELib can read all the data of a UPolys object.

**Comment by penguin21** [comment]
Well i just need can print the output of the umodel propieties like T3D in UDK if you can do it
